test/Window/Cursor.test.cpp

```diff
@@ -1,8 +1,37 @@
 #include <SFML/Window/Cursor.hpp>
 
+#include <doctest/doctest.h>
+
+#include <WindowUtil.hpp>
+#include <array>
 #include <type_traits>
 
 static_assert(!std::is_copy_constructible_v<sf::Cursor>);
 static_assert(!std::is_copy_assignable_v<sf::Cursor>);
 static_assert(!std::is_nothrow_move_constructible_v<sf::Cursor>);
 static_assert(!std::is_nothrow_move_assignable_v<sf::Cursor>);
+
+TEST_CASE("[Window] sf::Cursor" * doctest::skip(skipDisplayTests))
+{
+    SUBCASE("Construction")
+    {
+        const sf::Cursor cursor;
+    }
+
+    SUBCASE("loadFromPixels()")
+    {
+        sf::Cursor                  cursor;
+        std::array<std::uint8_t, 4> pixels{};
+
+        CHECK(!cursor.loadFromPixels(nullptr, {}, {}));
+        CHECK(!cursor.loadFromPixels(pixels.data(), {0, 1}, {}));
+        CHECK(!cursor.loadFromPixels(pixels.data(), {1, 0}, {}));
+        CHECK(cursor.loadFromPixels(pixels.data(), {1, 1}, {}));
+    }
+
+    SUBCASE("loadFromSystem()")
+    {
+        sf::Cursor cursor;
+        CHECK(cursor.loadFromSystem(sf::Cursor::Hand));
```

Comment:
What about testing all the cursors that are defined to be available on all platforms? 🤔

Reply:
I suppose I can try. I can at least test more than this one.